lib/ddtrace/configuration/settings.rb

```diff
@@ -55,6 +56,18 @@ class Settings
         end
       end
 
+      settings :debug do
```

Comment:
I'd prefer to not label as "debug", since these should be enabled by default (in the future) and aren't only used for debugging problems.

Reply:
Okay, need an alternate name then. `telemetry`? `diagnostics`? `

Reply:
```ruby
settings :health do
  option :metrics 
end
```

😆 

`diagnostics` seems ok to me

Reply:
Haha, only reason I don't like `health` is because I think if the name is to be a category, it should be broad enough to potentially hold other features. `diagnostics` seems more suitable as a category name, as `health` seems narrow.

I will have to change all the namespacing for consistency though, too.